lib/config/presets/internal/monorepo.ts

```diff
@@ -53,6 +53,7 @@ const repoGroups = {
   tsoa: 'https://github.com/lukeautry/tsoa',
   'typescript-eslint': 'https://github.com/typescript-eslint/typescript-eslint',
   'typography-js': 'https://github.com/KyleAMathews/typography.js',
+  'wordpress gutenberg': 'https://github.com/WordPress/gutenberg',
```

Comment:
We already have something that mentions `wordpress`, does that cause problems with adding the `wordpress gutenberg` monorepo?

```ts
const patternGroups = {
  babel6: '^babel6$',
  clarity: ['^@cds/', '^@clr/'],
  wordpress: '^@wordpress/',
  angularmaterial: ['^@angular/material', '^@angular/cdk'],
  'aws-java-sdk': '^com.amazonaws:aws-java-sdk-',
  'aws-java-sdk-v2': '^software.amazon.awssdk:',
  embroider: '^@embroider/',
  fullcalendar: '^@fullcalendar/',
};
```

Reply:
Thanks! I missed that completely.. I think this is not needed then, will close the PR